src/systems.rs

```diff
@@ -140,3 +169,17 @@ pub fn process_action_diffs<A: Actionlike, ID: Eq + Component + Clone>(
         }
     }
 }
+
+/// Release all inputs if [`DisableInput`] was added
```

Comment:
```suggestion
/// Release all actions in all [`ActionState`]s if [`DisableInput`] was added
```
It's important to be clear about the distinction between inputs and actions here.